nilearn/decoding/space_net.py

```diff
@@ -782,6 +771,17 @@ def __sklearn_tags__(self):
         tags.input_tags = InputTags(niimg_like=True, surf_img=False)
         return tags
 
+    @property
+    def _is_classification(self) -> bool:
+        # TODO remove for sklearn>=1.6
+        # this private method can probably be removed
+        # when dropping sklearn>=1.5 and replaced by just:
+        #   self.__sklearn_tags__().estimator_type == "classifier"
+        if SKLEARN_LT_1_6:
+            # TODO remove for sklearn>=1.6
+            return self._estimator_type == "classifier"
+        return self.__sklearn_tags__().estimator_type == "classifier"
+
```

Comment:
I think we do not need this function in the base class. The classification class should make necessary checks itself.